doc/pubreg.md

```diff
@@ -117,3 +118,15 @@ Note that key deletion is irreversible.
 Also note that due to how Maskinporten works, the expiration date of every
 existing key will be updated to today's date when creating a new key. Digdir is
 looking into a fix for this issue.
+
+## Viewing a client's audit log
```

Comment:
Bør vi ha med en liten merknad om at det ikke er mulig å hente ut logg for slettede klienter, og hvem man ev. kan kontakte?